src/material-experimental/mdc-chips/chip-row.ts

```diff
@@ -70,9 +70,13 @@ export class MatChipRow extends MatChip implements AfterContentInit, AfterViewIn
     super.ngAfterContentInit();
 
     if (this.removeIcon) {
-      // removeIcon has tabIndex 0 for regular chips, but should only be focusable by
-      // the GridFocusKeyManager for row chips.
-      this.removeIcon.tabIndex = -1;
+      // Defer setting the value in order to avoid the "Expression
+      // has changed after it was checked" errors from Angular.
```

Comment:
Could this be done earlier in the component lifecycle to avoid the extra round of change detection?

Reply:
I think the earliest it can be done is in ngAfterContentInit, because the removeIcon is a ContentChild. https://github.com/angular/components/blob/52dddcc64b353213f8ef106a505e53516d5c6093/src/material-experimental/mdc-chips/chip.ts#L230

Another possible fix would be injecting the parent chip into the remove icon and setting the tabIndex property in the remove icon's constructor. We'd have to make an injection token, or put the icons back in the chip file, like how they are in the non-mdc chips:
https://github.com/angular/components/blob/52dddcc64b353213f8ef106a505e53516d5c6093/src/material/chips/chip.ts#L401